src/Multicall.sol

```diff
@@ -1,17 +1,13 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.12;
 
+import {IMulticall} from "./interfaces/IMulticall.sol";
```

Comment:
Thanks for this PR! I'd prefer to leave the Multicall contracts untouched and just duplicate the structs in the interface files. Although this isn't conventionally best practice, I want the `src` contracts on `master` to match what's already deployed/verified—and have future deployments match existing deployments—which means leaving the `src` files untouched. (The resulting bytecode _should_ be identical regardless, but verification would look different + solc can be unintuitive sometimes + metadata hash would change anyway)

Reply:
Fair enough. I have pushed a new commit to undo the changes to the `src` contracts.